setup.py

```diff
@@ -0,0 +1,28 @@
+from setuptools import setup
+
+setup(
+    name="pvacseq",
+    version="1.0.1",
+    packages=["pvacseq", "pvacseq.lib"],
+    entry_points={
+        "console_scripts":[
+            "pvacseq = pvacseq.pvacseq:main"
+        ]
+    },
+    classifiers=[
+        'Development Status :: 4 - Beta',
+
+        'Intended Audience :: Science/Research',
+        'Topic :: Scientific/Engineering :: Bio-Informatics',
+
+        "Programming Language :: Python :: 3.5"
+    ],
+
+    author = "Griffith Lab",
```

Comment:
This should be updated to "Jasreet Hundal, Susanna Kiwala, Aaron Graubert, Jason Walker, Chris Miller, Malachi Griffith and Elaine Mardis"